pkg/controller/operators/olm/operator.go

```diff
@@ -1718,6 +1712,10 @@ func (a *Operator) transitionCSVState(in v1alpha1.ClusterServiceVersion) (out *v
 
 	// Check for intersecting provided APIs in intersecting OperatorGroups
 	allGroups, err := a.lister.OperatorsV1().OperatorGroupLister().List(labels.Everything())
+	if err != nil {
+		logger.WithError(err).Warn("failed to list operatorgroups")
+		return
+	}
```

Comment:
Note: need to double-check whether this is the correct behavior. It wasn't clear to me poking around whether I should be returning a syncError here.